src/endpoints/tv-shows.ts

```diff
@@ -115,7 +136,7 @@ export class TvShowsEndpoint extends BaseEndpoint {
     );
   }
 
-  async similar(id: number, options?: PageOption): Promise<SimilarTvShows> {
+  async similar(id: number, options?: LanguageOption & PageOption): Promise<SimilarTvShows> {
```

Comment:
Should this be combined into it's own options type instead of using an intersection (applies to all cases)?

Reply:
There were several places where this was done. I'd argue that, as  long as it stays short, it's easier to update options by composing them rather than having a type or an interfaced declare in another file.